WordPress/Classes/Models/PlanFeature.swift

```diff
@@ -0,0 +1,14 @@
+import Foundation
+import CoreData
+
+public class PlanFeature: NSManagedObject {
+
+    @nonobjc public class func fetchRequest() -> NSFetchRequest<PlanFeature> {
```

Comment:
If we got rid of the other one, could this one go too?

Reply:
Doh!  I should have just cleaned these up too.  Thanks for the nudge!